src/data_warehouse/cli/monitor.py

```
@@ -3,146 +1,219 @@
+        
+        if model:
+            where_clauses.append("model_name LIKE :model")
+            params['model'] = f"%{model}%"
+        
+        if test:
+            where_clauses.append("test_name LIKE :test")
+            params['test'] = f"%{test}%"
+        
+        if where_clauses:
+            query += "WHERE " + " AND ".join(where_clauses)
+        
```

Ruff (0.8.2)

Blank line contains whitespace

Remove whitespace from blank line

(W293)